src/main/java/de/srendi/advancedperipherals/common/util/inventory/FluidFilter.java

```diff
@@ -19,7 +24,9 @@
 import java.util.Map;
 
 //TODO tag
```

Comment:
so what's the TODO here 🤔

Reply:
from the initial 1.21 port where I didn't implemented tag filtering
Forgot to remove it haha